data/brands/shop/mobile_phone.json

```diff
@@ -1380,16 +1380,35 @@
     {
       "displayName": "小米",
       "id": "xiaomi-a9769c",
-      "locationSet": {"include": ["cn"]},
+      "locationSet": {"include": ["cn"], "exclude": ["hk", "mo"]},
       "tags": {
         "brand": "小米",
         "brand:en": "Xiaomi",
         "brand:wikidata": "Q1636958",
         "brand:wikipedia": "zh:小米集團",
-        "brand:zh": "小米之家",
+        "brand:zh": "小米",
         "name": "小米之家",
-        "name:en": "Xiaomi",
+        "name:en": "Mi",
+        "name:zh": "小米之家",
+        "shop": "mobile_phone"
+      }
+    },
+    {
+      "displayName": "小米 Xiaomi",
+      "locationSet": {"include": ["hk", "mo"]},
+      "tags": {
+        "brand": "小米 Xiaomi",
+        "brand:en": "Xiaomi",
+        "brand:wikidata": "Q1636958",
+        "brand:wikipedia": "zh:小米集團",
+        "brand:zh": "小米",
```

Comment:
My opinion is that if `brand:zh-Hans` and `brand:zh-Hant` are exactly same, I suggest that we only reserve `brand:zh`

Reply:
Yes, we can do this